entries/SWC-137.md

```diff
@@ -0,0 +1,20 @@
+# Title
+Message call with hardcoded gas amount
+
+## Relationships
+[CWE-655: Improper Initialization](https://cwe.mitre.org/data/definitions/665.html)
```

Comment:
Is this more accurate https://cwe.mitre.org/data/definitions/664.html?

Reply:
I didn't find anything that really fit... by "resource" they're referring to Java class instances and things like that. Do you think that's more appropriate than using the broad category?